{"schemaVersion":{"patch":0,"major":0,"minor":3},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"error","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:6OpenAI8APIErrorV","text":"APIError","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","kind":"typeIdentifier"}],"platforms":["macOS"]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/openai\/apierrorresponse\/error"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","sections":[],"metadata":{"symbolKind":"property","roleHeading":"Instance Property","modules":[{"name":"OpenAI"}],"externalID":"s:6OpenAI16APIErrorResponseV5errorAA0C0Vvp","role":"symbol","fragments":[{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"error","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:6OpenAI8APIErrorV","text":"APIError","kind":"typeIdentifier"}],"title":"error"},"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error"},"references":{"doc://OpenAI/documentation/OpenAI":{"abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","url":"\/documentation\/openai","kind":"symbol","type":"topic","title":"OpenAI","role":"collection"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/error":{"url":"\/documentation\/openai\/apierrorresponse\/error","kind":"symbol","abstract":[],"title":"error","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"error"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:6OpenAI8APIErrorV","text":"APIError","kind":"typeIdentifier"}],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error","type":"topic","role":"symbol"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse":{"url":"\/documentation\/openai\/apierrorresponse","kind":"symbol","abstract":[],"title":"APIErrorResponse","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"APIErrorResponse","kind":"identifier"}],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse","navigatorTitle":[{"text":"APIErrorResponse","kind":"identifier"}],"type":"topic","role":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError":{"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"text":"APIError","kind":"identifier"}],"abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","title":"APIError","url":"\/documentation\/openai\/apierror","navigatorTitle":[{"text":"APIError","kind":"identifier"}],"kind":"symbol","type":"topic","role":"symbol"}}}